modules/dag-shared/src/main/scala/org/tessellation/dag/block/processing/BlockAcceptanceManager.scala

```diff
@@ -58,7 +58,7 @@ object BlockAcceptanceManager {
           toProcess: List[(Signed[DAGBlock], TxChains)]
         ): F[BlockAcceptanceState] =
           for {
-            currState <- toProcess.foldLeftM(initState) { (acc, blockAndTxChains) =>
+            currState <- toProcess.foldLeftM(initState.copy(awaiting = List.empty)) { (acc, blockAndTxChains) =>
```

Comment:
Great that you've found it. It was like that initially, but I mistakenly changed it. Thanks for fixing it.